apps/frontend/package.json

```diff
@@ -6,7 +6,7 @@
     "node": "12.x"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "TSC_COMPILE_ON_ERROR=true react-scripts start",
```

Comment:
The TSC_COMPILE_ON_ERROR is mandatory everytime? If that is the case can we use something like https://www.npmjs.com/package/cross-env?

Reply:
fixed, thanks! :+1: